### ISSUE TITLE
Regarding the issue of _ reward_foot_slip

### ISSUE THREAD

**Issue by Chen-MP** [engineai-robotics/engineai_legged_gym#6]
def _reward_foot_slip(self):    
        contact = self.contact_forces[:, self.feet_indices, 2] > 5.
        # TODO: 我认为应该是 线速度 vx, vy 应该是在 7:10 ； wx, wy 在 10:12
        #              所以为什么不是线速度而是角速度呢
        foot_speed_norm = torch.norm(self.rigid_state[:, self.feet_indices, 10:12], dim=2)
        rew = torch.sqrt(foot_speed_norm)
        rew *= contact
        return torch.sum(rew, dim=1)
